src/main/scala/is/hail/linalg/BlockMatrix.scala

```diff
@@ -148,6 +149,149 @@ object BlockMatrix {
     }
   }
 
+  // rectangles checked in Python
+  def exportRectangles(
+    hc: HailContext,
+    input: String,
+    output: String,
+    flattenedRectangles: Array[Long],
+    delimiter: String,
+    nPartitions: Int): Unit = {
+    require(flattenedRectangles.length % 4 == 0)
+
+    val hadoop = hc.sc.hadoopConfiguration
+    if (!hadoop.exists(input + "/_SUCCESS"))
+      fatal("Write failed: no success indicator found")
```

Comment:
maybe put something in here about the input path? This message implies that something went wrong inside exportRectangles, not upstream.

Reply:
good idea, i also pulled this check into its own function.